### ISSUE TITLE
Ambilight not working

### ISSUE THREAD

**Issue by drford17** [zim514/script.service.hue#297]
Hi,

I configured Hue Service addon and I was able to configure the bridge. 
When I am starting the playback I get the following error:

```
2024-09-28 01:27:17.745 T:1797    error <general>: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: <class 'AttributeError'>
                                                   Error Contents: 'NoneType' object has no attribute 'getDuration'
                                                   Traceback (most recent call last):
                                                     File "/storage/.kodi/addons/script.service.hue/resources/lib/ambigroup.py", line 81, in onAVStarted
                                                       if self.activation_check.validate():
                                                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                                                     File "/storage/.kodi/addons/script.service.hue/resources/lib/lightgroup.py", line 329, in validate
                                                       if self._is_within_schedule() and self._video_activation_rules():
                                                                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                                                     File "/storage/.kodi/addons/script.service.hue/resources/lib/lightgroup.py", line 202, in _video_activation_rules
                                                       duration = info_tag.getDuration() / 60
                                                                  ^^^^^^^^^^^^^^^^^^^^
                                                   AttributeError: 'NoneType' object has no attribute 'getDuration'
                                                   -->End of Python script error report<--
```

Please also find the system information below:
Hue Service addons is running on latest version v2.0.11

2024-09-28 01:26:31.216 T:1722     info <general>: Starting Kodi (22.0-ALPHA1 (21.90.700) Git:cdfc755adde418d33072a98b764c7b631b9e5cfb). Platform: Linux x86 64-bit
2024-09-28 01:26:31.216 T:1722     info <general>: Using Release Kodi x64
2024-09-28 01:26:31.216 T:1722     info <general>: Kodi compiled 2024-09-05 by GCC 14.2.0 for Linux x86 64-bit version 6.10.3 (395779)
2024-09-28 01:26:31.216 T:1722     info <general>: Running on LibreELEC (heitbaum): devel-20240905134548-36bff65 13.0, kernel: Linux x86 64-bit version 6.10.3
2024-09-28 01:26:31.216 T:1722     info <general>: FFmpeg version/source: 6.0.1
2024-09-28 01:26:31.216 T:1722     info <general>: Host CPU: Intel(R) Core(TM) Ultra 9 185H, 22 cores available

Thank you

**Comment by drford17** [comment]
fixed by downgrading to 2.0.10

**Comment by zim514** [comment]
Thanks for the logs! This should be fixed in v2.0.12
